src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -1,18 +1,30 @@
 from __future__ import annotations  # for type hints
 
+import ast
+import logging
 from abc import ABC, abstractmethod
 from ast import parse as parse_python
-from collections.abc import Callable
+from collections.abc import Callable, Iterable
 from enum import Enum
 
 from sqlglot import ParseError as SQLParseError
 from sqlglot import parse as parse_sql
 from databricks.sdk.service.workspace import Language
 
-NOTEBOOK_HEADER = " Databricks notebook source"
-CELL_SEPARATOR = " COMMAND ----------"
-MAGIC_PREFIX = ' MAGIC'
-LANGUAGE_PREFIX = ' %'
+from databricks.labs.ucx.source_code.astlinter import ASTLinter
+from databricks.labs.ucx.source_code.base import Linter, Advice, Advisory
+
+
+logger = logging.getLogger(__name__)
+# use a specific logger for sqlglot warnings so we can disable them selectively
+sqlglot_logger = logging.getLogger(f"sqlglot in {__name__}")
```

Comment:
Python loggers use a hierarchy with dots to configure (subsets of) loggers, maybe do the following:

```suggestion
sqlglot_logger = logging.getLogger(f"{__name__}.sqlglot")
```